### DISCUSSION TITLE
Configure GitHub Action to automatically publish alpha latter snapshot versions?

### DISCUSSION THREAD

**Discussion by StringKe** [fuma-nama/fumadocs#944] [General]
The code in examples does not match the current release and the project cannot be created and run correctly.

**Comment by fuma-nama** [comment]
Do you mean the `/examples` folder in the repo? See `main` branch for the current version

**Reply by StringKe** [reply]
I would like to use the latest version of the code repository directly.

**Reply by fuma-nama** [reply]
You can build from source by cloning the repo
